src/DataAPI.jl

```diff
@@ -100,6 +105,26 @@ function levels(x)
     levs
 end
 
+"""
+    isordered(A)
+
+Test whether entries in `A` can be compared using `<`, `>` and similar operators,
+using the ordering of `levels(A)`.
+
+If a type of `A` implements `levels` with a meaningful order then it should also
```

Comment:
@nalimilan - could you please comment in the implementation how your proposal should be incorporated? Thank you!